### PR TITLE
Separate build functions

### PR DESCRIPTION
**Pull request by MartinPetkov**
This way they can be individually used in release automation.

### CODE REVIEW

File: build/build-binaries.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# Helper to build release artifacts - binaries only.
+#
+# Usage (from repo root):
+# ./build/build-binaries.sh -o . -v v0.1.0
+
+# Collects args and defines build functions.
+source "$(dirname "${0}")/build-include.sh"
+
+# Build binaries only.
+build_binaries
```
Comment (umairidris) [inline build/build-binaries.sh:26]: Can we use something like a make file and user can just say what they want to build?

e.g.

make binaries v0.1.0
make templates v.02.0 (append templates- automatically)

Comment (MartinPetkov) [reply build/build-binaries.sh:26]: From what I can tell, you're not really supposed to be calling make with arguments, only targets.

Comment (MartinPetkov) [reply build/build-binaries.sh:26]: I'll see if there's a sensible way to do it with environment variables or something.

Comment (umairidris) [reply build/build-binaries.sh:26]: No wories then, this is fine

**Pull request merged**